### PR TITLE
LPS-156602 [PRM] MDF Claim - Due Dates and Notification

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-cron/src/main/java/com/liferay/partner/PartnerCommandLineRunner.java [hunk 1]
```diff
@@ -106,6 +106,76 @@ public void run(String... args) throws Exception {
...
+		if (responseJSONObject.getInt("totalCount") > 0) {
+			JSONArray itemsJSONArray = responseJSONObject.getJSONArray("items");
+
+			for (int i = 0; i < itemsJSONArray.length(); i++) {
+				JSONObject itemJSONObject = itemsJSONArray.getJSONObject(i);
+
+				String activityErc = itemJSONObject.getString(
+					"r_actToMDFClmActs_c_activityERC");
+
+				JSONObject activityJSONArray = itemJSONObject.getJSONObject(
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-cron/src/main/java/com/liferay/partner/PartnerCommandLineRunner.java:135]:
```suggestion
				JSONObject activityJSONObject = itemJSONObject.getJSONObject(
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:test:relevant

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:test:relevant

**Comment by thaynam** [comment]
ci:test:relevant

**Comment by thaynam** [comment]
ci:test:stable

**Pull request closed without merging**